staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -213,3 +224,48 @@ func (f *FieldManager) buildManagerInfo(prefix string, operation metav1.ManagedF
 	}
 	return internal.DecodeManager(&managerInfo)
 }
+
+// StripFields that don't contribute much value to an objects managedFields
+func (f *FieldManager) StripFields(obj runtime.Object, typed typed.TypedValue, apiVersion fieldpath.APIVersion, managed fieldpath.ManagedFields, manager string) error {
```

Comment:
You only seem to call this from within the `FieldManager`, which I think makes sense, so it should probably be private.
Also I'm assuming this is changing manager in place, maybe that should be said in the doc? Also is this changing anything but `managed[manager]` can it take this in place (and maybe return it?)